lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -301,6 +303,7 @@ follow nl tr cps yield rollback header =
                 logError tr $ "Non-recoverable error following the chain: " <> eT
             _ -> do
                 logError tr $ "Recoverable error following the chain: " <> eT
+                logError tr $ "Sleeping " <> (T.pack . show $ retryDelay delay)
```

Comment:
I forgot to remove this line, but maybe we do want to include the delay in the first error?

Reply:
Removed.